tasks.rb

```diff
@@ -422,6 +423,7 @@ def create_hpxmls
     'base-schedules-simple-vacancy-year-round.xml' => 'base.xml',
     'base-schedules-detailed-all-10-mins.xml' => 'base-simcontrol-timestep-10-mins.xml',
     'base-schedules-detailed-occupancy-stochastic.xml' => 'base.xml',
+    'base-schedules-detailed-occupancy-stochastic-residents-0.xml' => 'base.xml',
```

Comment:
I don't understand the point of this HPXML file. If there are no occupants, then by definition there are no stochastic schedules. What is it trying to show/test?

Reply:
That `occupancy-stochastic-residents-0.csv` does not get created (looks like `update_hpxmls` needed to be run again, which addresses your comment below).

Reply:
So can we delete this HPXML or does it serve a purpose? You already wrote a unit test to ensure that the BuildResScheduleFile measure handles zero occupants correctly.

Reply:
Deleted.